clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -12,7 +12,7 @@
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/eclipse/che-che4z-lsp-for-cobol.git"
+        "url": "git@github.com:grianbrcom/che-che4z-lsp-for-cobol.git"
```

Comment:
Why do you use ssh here?

Reply:
Good catch. I'm used to using ssh. I'll check the work with https.

Reply:
The plugin works well with https.